app/scripts/controllers/transactions.js

```diff
@@ -26,7 +26,7 @@ angular.module('openhimConsoleApp')
     $scope.settings.list = {};
     $scope.settings.list.tabview = 'same';
     $scope.settings.filter = {};
-    $scope.settings.filter.limit = 10;
+    $scope.settings.filter.limit = 100;
```

Comment:
I still think 10 is too little, but yeah, that's a good a point about seeing the buttons at the end, so yeah, it's probably good to stick to it then. I'll just have to change my user settings :)